<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngineeringCalculator</class>
 <widget class="QMainWindow" name="EngineeringCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EngineeringCalculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: rgb(216, 255, 216);  /* Бледно-зелёный фон */
}

QPushButton {
    background-color: #FFD8A6;            /* Бледно-оранжевый */
    color: #5A4A3A;                      /* Тёмно-бежевый текст */
    border: 1px solid #E0B080;           /* Граница оранжевого оттенка */
    border-radius: 5px;
    font-size: 16px;
    min-width: 40px;
    min-height: 40px;
    font-weight: 500;                    /* Полужирный текст */
}

QPushButton:hover {
    background-color: #FFC080;           /* Более насыщенный оранжевый */
    border: 1px solid #D0A070;
}

QPushButton:pressed {
    background-color: #FFB366;           /* Ещё более тёмный оранжевый */
    border: 1px solid #C09060;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>42</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Equals">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>211</y>
      <width>42</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeSignButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Subtract">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Divide">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Multiply">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Display">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="Button8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button0">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>Sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>Cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Log">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sqrt">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Power">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReturnButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>131</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>Вернуться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDot">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPi">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>π</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonE">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QListWidget" name="historyList">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
